README.md

```diff
@@ -132,7 +110,7 @@ All packages must contain the following scripts to perform the stated actions:
 | :-----------: | ------------------ | :--------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 |      ✅       | `start`            | Runs complete test and build suite and starts the dev server.                                                                                                          |
 |      ✅       | `serve`            | Runs build suite and starts the dev server. When serving `core`, takes optional `open` value as argument to open specific page. (e.g: `npm run serve core --open=bot`) |
-|      ✅       | `build`            | Runs build suite and outputs the result into `dist`. Takes optional `base` value as argument.                                                                          |
+|      ✅       | `build:all`        | Runs build suite and outputs the result into `dist`. Takes optional `base` value as argument.                                                                          |
```

Comment:
we have another build script `build:one` which builds just the passed workspace name
for instance `npm run build:one cashier`

Reply:
it is updated ✅